### PR TITLE
document deploys

### PR DESCRIPTION
**Pull request by Fil**

### CODE REVIEW

File: docs/deploying.md [hunk 1]
```diff
@@ -54,7 +54,25 @@ If the specified config file does not yet exist, you will again be prompted to c
...
+Once the app has been set up, you can have Observable build the next deploys for you. You can opt in to build *on commit* (whenever you push a new commit to your project’s default branch), and *on schedule*, such as daily or weekly.
+
+Automatic deploys — also known as *continuous deployment* or *CD* — ensure that your  data is up to date, and that any changes you make to your app are immediately reflected in the deployed version.
+
+On your project’s configuration page on Observable, open the **Build settings** tab and click **Connect to GitHub**. This will prompt you to sign in to GitHub and authorize Observable to access (a selection of) your repositories. Once you’ve done that, you can select the project repository and branch you want to deploy from. Observable will then listen for changes in the repo and deploy the app automatically.
+
+In the same tab, you can choose the **Build environment** — “Observable Cloud” is the default, but Enterprise customers can opt in to [build on their own infrastructure](https://observablehq.com/documentation/projects/security#on-premises-hosting-self-hosting).
```
Comment (tophtucker) [inline docs/deploying.md:63]: i think the best link for build environment is https://observablehq.com/documentation/data-apps/deploys#build-environments

& best link for on-premises is https://observablehq.com/documentation/data-apps/security#on-premises-environments

File: docs/deploying.md [hunk 2]
```diff
@@ -54,7 +54,25 @@ If the specified config file does not yet exist, you will again be prompted to c
 
 ## Automated deploys
 
-To set up automatic deploys (also known as *continuous deployment* or *CD*), we recommend [GitHub Actions](https://github.com/features/actions). In your git repository, create and commit a file at `.github/workflows/deploy.yml`. Here is a starting example:
+Once the app has been set up, you can have Observable build the next deploys for you. You can opt in to build *on commit* (whenever you push a new commit to your project’s default branch), and *on schedule*, such as daily or weekly.
+
+Automatic deploys — also known as *continuous deployment* or *CD* — ensure that your  data is up to date, and that any changes you make to your app are immediately reflected in the deployed version.
+
+On your project’s configuration page on Observable, open the **Build settings** tab and click **Connect to GitHub**. This will prompt you to sign in to GitHub and authorize Observable to access (a selection of) your repositories. Once you’ve done that, you can select the project repository and branch you want to deploy from. Observable will then listen for changes in the repo and deploy the app automatically.
```
Comment (tophtucker) [inline docs/deploying.md:61]: i think those whole paragraph could be cut; you’ve covered all you need to when you say “on commit” above

File: docs/deploying.md [hunk 3]
```diff
@@ -54,7 +54,25 @@ If the specified config file does not yet exist, you will again be prompted to c
...
+
+In the same tab, you can choose the **Build environment** — “Observable Cloud” is the default, but Enterprise customers can opt in to [build on their own infrastructure](https://observablehq.com/documentation/projects/security#on-premises-hosting-self-hosting).
+
+To set up a schedule, follow the instructions in the **Automation** tab.
```
Comment (tophtucker) [inline docs/deploying.md:65]: could cut this, already covered by first line

File: docs/deploying.md (hunk 2 above)
Comment (tophtucker) [reply docs/deploying.md:61]: i’ve been saying “data app settings page” or just “app settings” 🤷

File: docs/deploying.md [hunk 4]
```diff
@@ -54,7 +54,25 @@ If the specified config file does not yet exist, you will again be prompted to c
...
+
+If your project needs secrets (such as API keys and database connection credentials), you can add them in the **Secrets** tab. These values are available as environment variables to be used, typically, by data loaders. They are redacted in logs, and are not present in the deployed app.
+
+You can also deploy manually at any time from the project configuration page, for example to refresh the data without waiting for the next scheduled deploy.
```
Comment (tophtucker) [inline docs/deploying.md:69]: could cut this, not that important

File: docs/deploying.md [hunk 5]
```diff
@@ -54,7 +54,25 @@ If the specified config file does not yet exist, you will again be prompted to c
...
+
+The build logs are then available from the **History** tab. If a build fails, you can see the error message to help you debug the issue.
```
Comment (tophtucker) [inline docs/deploying.md:71]: most of the time i'm just looking at the logs for the latest deploy so i'm using "View logs" in the deploy status. i guess again you could just say that you can see logs w/o the detail of telling ppl where it is.

**Review by tophtucker — COMMENTED** [review]
this is a great compact summary of functionality and stands well on its own! 

given the existence of the other docs, i guess i agree that this could be even shorter. https://observablehq.com/documentation/data-apps/deploys is probably the best page in those docs to link to

here, it feels less important to tell people where each button is and more important to cover the general functionality. e.g. it feels more important to say "you can add secrets (such as API keys and database connection credentials) that will be passed as environment variables, typically to be used by data loaders", and less important to say what tab it's on.

**Review by mbostock — APPROVED** [review]

**Pull request merged**